resources/wikia/modules/krux.js

```diff
@@ -0,0 +1,64 @@
+/*global Krux,define*/
+window.Krux || ((Krux = function () {
+	Krux.q.push(arguments);
+}).q = []);
+
+define('wikia.krux', ['wikia.window'], function (win) {
+	'use strict';
+
+	var maxNumberOfKruxSegments = 27;
+
+	function load(confid) {
```

Comment:
Me neither. I guess @dianafa experimented with it (and @hakubo with 
similar issues for ComScore and QuantCast), so let's consult them!

On 03/12/2015 01:19 PM, Igor Rogatty wrote:
> In resources/wikia/modules/krux.js
> <https://github.com/Wikia/app/pull/6605#discussion_r26296771>:
>
>> @@ -0,0 +1,64 @@
>> +/*global Krux,define*/
>> +window.Krux || ((Krux = function () {
>> +	Krux.q.push(arguments);
>> +}).q = []);
>> +
>> +define('wikia.krux', ['wikia.window'], function (win) {
>> +	'use strict';
>> +
>> +	var maxNumberOfKruxSegments = 27;
>> +
>> +	function load(confid) {
>
> I don't know Krux internals and if we have to load script on every
> article view, but...
> We don't want to append another |<script>| tag on every article view,
> now do we? They would accumulate.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/app/pull/6605/files#r26296771>.
>